### PR TITLE
Integrate Ent diff into OS

### PR DESCRIPTION
**Pull request by ryaplots**
#### Summary

This PR merges unnecessary diff between Enterprise and OS. Refers to [#3051](https://github.com/TheThingsIndustries/lorawan-stack/issues/3051)

#### Testing

e2e and manually.

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [ ] Documentation: Relevant documentation is added or updated.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: pkg/webui/console/components/gateway-data-form/index.js
Comment (bafonins) [inline pkg/webui/console/components/gateway-data-form/index.js:170]: I dont think we need this in OS since there is no claiming view. Basiclaly, `console/gateways/claim` is not reachable.

**Review by bafonins — APPROVED** [review]
See my comment. the rest looks good

File: pkg/webui/console/components/gateway-data-form/index.js
Comment (kschiffer) [reply pkg/webui/console/components/gateway-data-form/index.js:170]: Indeed. Until we figured out what to do here, we should change the message to

> It appears like you are trying to register a \<b>The Things Indoor Gateway\</b>. Such gateways are already preregistered and need to be claimed instead. Claiming can only be performed on a deployment that supports gateway claiming.

File: pkg/webui/components/notification/index.js
Comment (kschiffer) [inline pkg/webui/components/notification/index.js:82]: `action` is actually still used here `console/components/webhook-form/index.js`

Make sure you refactor that before merging. This should also mean that this crashes in Enterprise.

**Review by kschiffer — CHANGES_REQUESTED** [review]

File: pkg/webui/components/notification/index.js
Comment (ryaplots) [reply pkg/webui/components/notification/index.js:82]: Will resolve this for Enterprise in another PR

**Review by kschiffer — APPROVED** [review]
LGTM, please only merge after incorporating https://github.com/TheThingsIndustries/lorawan-stack/pull/3067

**Review by adriansmares — APPROVED** [review]

**Pull request merged**
